modules/showheroes-bsBidAdapter.js

```diff
@@ -250,12 +166,13 @@ function createBids(bidRes, reqData) {
     let bidUnit = {};
     bidUnit.cpm = bid.cpm;
     bidUnit.requestId = requestId;
-    bidUnit.adUnitCode = reqBid.adUnitCode;
+    bidUnit.adUnitCode = bid.adUnitCode;
     bidUnit.currency = bid.currency;
     bidUnit.mediaType = bid.mediaType || VIDEO;
-    bidUnit.ttl = TTL;
+    bidUnit.ttl = bid.exp || TTL;
     bidUnit.creativeId = 'c_' + requestId;
```

Comment:
Most probably `creativeId` is not needed. My guess was it was needed for the previous renderer, but since that is no longer working it's not needed. But then again, on the other hand, we don't have a renderer in place, so at this point, we can't be really sure what is really needed and what isn't.

`advertiserDomains` is something each bidder must provide in response, it's a domain of advertisers delivering the ad.
It's being passed here: https://github.com/showheroes/viralize/blob/PLT-2267-adserver-add-dynamic-price-for-prebid-integration/viralize/web/executors/ad/auction.py#L131